bloom/logging.py

```diff
@@ -60,7 +55,7 @@
 
 _emoji_check_mark = "✅  "
 _emoji_cross_mark = "❌  "
-_is_mac_lion_or_greater = parse_version(mac_ver()[0]) >= parse_version('10.7.0')
+_is_mac = (mac_ver()[0] != '')
```

Comment:
In general I'm fine with dropping "support" for Mac OS Lion+, let's make sure that's a full bump for the next release though.

+ Snow Leopard was the last great release of Mac OS anyway.